documentation/tutorials/getting-started-c/corelib.md

```diff
@@ -53,7 +53,7 @@ void main(void) {
     SCREEN *screen;
 +   load_library("/lib/core");
     get_lcd_lock();
-    screen = screen_allocate();
```

Comment:
You do still need to allocate the screen, but I removed the line from that diff since it seemed cluttered. The point of that section is just to show load_library in context, not to duplicate the entire function.